internal/resources/snapshot_backup_schema.go

```diff
@@ -0,0 +1,59 @@
+package resources
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+)
+
+func SnapshotBackupSchema() schema.Schema {
+	return schema.Schema{
+		MarkdownDescription: "Manages snapshot backup resource",
+		Attributes: map[string]schema.Attribute{
+			"app_service":     WithDescription(stringAttribute([]string{computed, useStateForUnknown}), "The GUID4 ID of the app service."),
+			"id":              WithDescription(stringAttribute([]string{computed, useStateForUnknown}), "The GUID4 ID of the snapshot backup."),
+			"cluster_id":      WithDescription(stringAttribute([]string{required, requiresReplace}, validator.String(stringvalidator.LengthAtLeast(1))), "The GUID4 ID of the cluster."),
+			"project_id":      WithDescription(stringAttribute([]string{required, requiresReplace}, validator.String(stringvalidator.LengthAtLeast(1))), "The GUID4 ID of the project."),
+			"organization_id": WithDescription(stringAttribute([]string{required, requiresReplace}, validator.String(stringvalidator.LengthAtLeast(1))), "The GUID4 ID of the organization."),
+			"created_at":      WithDescription(stringAttribute([]string{computed, useStateForUnknown}), "The RFC3339 timestamp representing the time at which snapshot backup was created."),
+			"expiration":      WithDescription(stringAttribute([]string{computed}), "The RFC3339 timestamp representing the time at which snapshot backup will expire."),
+			"retention":       WithDescription(int64Attribute(optional, computed), "The retention of the snapshot backup in hours."),
+			"regions_to_copy": WithDescription(stringListAttribute(optional), "The regions to copy the snapshot backup to."),
```

Comment:
use set instead of a list

list is for an ordered collection, which I don't think is needed for this attribute